lib/reek/ast/sexp_extensions.rb

```diff
@@ -421,7 +422,7 @@ def name
         end
 
         def value
-          children.last
+          children.last if children.size > 2
```

Comment:
Or you can go with `children[2]` and avoid a lot of complication :smile:.

:+1: on adding the explanation and example.